modules/openapi-generator/src/main/resources/swift4/APIs.mustache

```diff
@@ -11,6 +11,7 @@ open class {{projectName}}API {
     public static var credential: URLCredential?
     public static var customHeaders: [String:String] = [:]
     public static var requestBuilderFactory: RequestBuilderFactory = AlamofireRequestBuilderFactory()
+    public static var apiResponseQueue: DispatchQueue = .main
```

Comment:
@syatyo Since now the Swift generator support the visibility modifier, this line should become
`{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} static var apiResponseQueue: DispatchQueue = .main`

Reply:
I've tried to fix it via https://github.com/OpenAPITools/openapi-generator/pull/4591/commits/404288da5713ceeb5546caf01507be59742161c4